firefox-ios/Shared/Strings.swift

```diff
@@ -3638,7 +3638,7 @@ extension String {
         key: "Mobile Bookmarks",
         tableName: "Storage",
         value: nil,
-        comment: "The legacy title of the folder that contains mobile bookmarks. This should match bookmarks.folder.mobile.label on Android.")
```

Comment:
This is a carry over from https://github.com/mozilla-mobile/firefox-ios/pull/25159#pullrequestreview-2664863646